vllm/utils.py

```diff
@@ -388,10 +389,16 @@ def get_distributed_init_method(ip: str, port: int) -> str:
     return f"tcp://[{ip}]:{port}" if ":" in ip else f"tcp://{ip}:{port}"
 
 
-def get_open_port(port: Optional[int] = None) -> int:
-    if port is None:
-        # Default behavior here is to return a port for multi-gpu communication
-        port = envs.VLLM_PORT
+def get_open_zmq_ipc_path() -> str:
+    base_rpc_path = envs.VLLM_RPC_BASE_PATH
+    if base_rpc_path is None:
+        base_rpc_path = "/tmp"
```

Comment:
it will never be `None`, because you use `os.getenv('VLLM_RPC_BASE_PATH', '/tmp')` above.